src/Components/MeditationStats.js

```diff
@@ -18,18 +14,43 @@ const getData = async () => {
 };
 
 const MeditationStats = () => {
-  const [userInfo, setUserInfo] = useState(null);
+  const [userEmail, setUserEmail] = useState("");
```

Comment:
Nice job figuring this out! Love the use of use effect triggered when the userEmail state variable changes to call the refetch.

Reply:
Nice, y'all!!